src/bundle.c

```diff
@@ -474,6 +474,9 @@ gboolean resign_bundle(RaucBundle *bundle, const gchar *outpath, GError **error)
 		goto out;
 	}
 
+	/* invalidate old keyring */
```

Comment:
This does not really reflect why this is needed. Maybe


```suggestion
	/* invalidate to not trigger post-signing verification with inappropriate keyring */
```

Reply:
"invalidate keyring to avoid post-signing verification with inappropriate keyring"?

Reply:
would accept it ;)

Reply:
Done.